spec/index.bs

```diff
@@ -13718,35 +13841,33 @@ partial interface GPUDevice {
         A [=stack=] of [=GPU error scopes=] that have been pushed to the {{GPUDevice}}.
 </dl>
 
-<div algorithm>
+<div algorithm data-timeline=device>
     The <dfn abstract-op>current error scope</dfn> for a {{GPUError}} |error| and {{GPUDevice}}
-    |device| is determined by issuing the following steps to the [=Device timeline=] of |device|:
+    |device| is determined by issuing the following steps to the [=device timeline=] of |device|:
 
-    <div data-timeline=device>
-        [=Device timeline=] steps:
+    [=Device timeline=] steps:
 
-        1. If |error| is an instance of:
+    1. If |error| is an instance of:
 
-            <dl class=switch>
-                : {{GPUValidationError}}
-                :: Let |type| be "validation".
-                : {{GPUOutOfMemoryError}}
-                :: Let |type| be "out-of-memory".
-                : {{GPUInternalError}}
-                :: Let |type| be "internal".
-            </dl>
-        1. Let |scope| be the last [=list/item=] of |device|.{{GPUDevice/[[errorScopeStack]]}}.
-        1. While |scope| is not `undefined`:
-            1. If |scope|.{{GPU error scope/[[filter]]}} is |type|, return |scope|.
-            1. Set |scope| to the previous [=list/item=] of
-                |device|.{{GPUDevice/[[errorScopeStack]]}}.
-        1. Return `undefined`.
-    </div>
+        <dl class=switch>
+            : {{GPUValidationError}}
+            :: Let |type| be "validation".
+            : {{GPUOutOfMemoryError}}
+            :: Let |type| be "out-of-memory".
+            : {{GPUInternalError}}
+            :: Let |type| be "internal".
+        </dl>
+    1. Let |scope| be the last [=list/item=] of |device|.{{GPUDevice/[[errorScopeStack]]}}.
+    1. While |scope| is not `undefined`:
+        1. If |scope|.{{GPU error scope/[[filter]]}} is |type|, return |scope|.
+        1. Set |scope| to the previous [=list/item=] of
+            |device|.{{GPUDevice/[[errorScopeStack]]}}.
+    1. Return `undefined`.
 </div>
 
 <div algorithm>
     To <dfn abstract-op lt="Dispatch error|dispatch error">dispatch an error</dfn> {{GPUError}}
-    |error| on {{GPUDevice}} |device|, run the following steps on the [=Device timeline=]
+    |error| on {{GPUDevice}} |device|, run the following steps on the [=device timeline=]
     of |device|:
```

Comment:
"dispatch an error" should only be called from the device timeline, so I think something like this..?

```
    To <dfn abstract-op lt="Dispatch error|dispatch error">dispatch an error</dfn> {{GPUError}}
    |error| run the following [=device timeline=] steps:

...
1. Let |device| be the device for the current [=device timeline=].
```

Reply:
Actually that's not necessary since the callers already have the device. Can just be something like
```suggestion
    |error| on {{GPUDevice}} |device|, run the following [=device timeline=] steps:
```